packit_service/worker/build/build_helper.py

```diff
@@ -373,9 +411,59 @@ def _report(
         The status reporting should be done through this method
         so we can extend it in subclasses easily.
         """
-        self.status_reporter.report(
-            description=description, state=state, url=url, check_names=check_names,
-        )
+        if (
+            isinstance(self.project, GitlabProject)
+            and not self.check_project_permission
```

Comment:
The name `check_project_permission` looks more like a method. With properties, I would use some `is_*` form of the name. E.g. `is_reporting_allowed`, or `is_fork_status_allowed`,...

Reply:
Yup, is_reporting_allowed is better name here. Changed it!